### PR TITLE
🌱 main.go: add flags for webhook cert-name and -key

### PR DESCRIPTION
**Pull request by chrischdi**
**What this PR does / why we need it**:

I diffed CAPI's main.go and CAPV's, found this diff in the flags.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -205,6 +205,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&webhookOpts.CertDir, "webhook-cert-dir", "/tmp/k8s-webhook-server/serving-certs/",
 		"Webhook cert dir, only used when webhook-port is specified.")
 
+	fs.StringVar(&webhookOpts.CertName, "webhook-cert-name", "tls.crt",
+		"Webhook cert name, only used when webhook-port is specified.")
```
Comment (sbueringer) [inline main.go:209]: > only used when webhook-port is specified

Is this true?

Comment (chrischdi) [reply main.go]: Removed. The port nowadays defaults to 9443, so its always set and defaults to tls.

File: main.go [hunk 2]
```diff
@@ -200,10 +200,16 @@ func InitFlags(fs *pflag.FlagSet) {
 		"Maximum number of queries that should be allowed in one burst from the cluster cache tracker clients to the Kubernetes API server of workload clusters.")
 
 	fs.IntVar(&webhookOpts.Port, "webhook-port", defaultWebhookPort,
-		"Webhook Server port")
+		"Webhook Server port.")
```
[the remark this answers is not in the document]
Comment (sbueringer) [reply main.go:203]: CR will default to 9443 if it's 0. Sounds reasonable to me to adjust our default value to make the defaulting more obvious

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Pull request merged**